Core/clim-core/gadgets.lisp

```diff
@@ -1659,13 +1661,22 @@ and must never be nil.")
 
 (defmethod compose-space ((pane slider-pane) &key width height)
   (declare (ignore width height))
-  (let ((minor (+ 8 (* 2 4) (if (gadget-show-value-p pane) 30 0)))
-        (major 128))
-    (if (eq (gadget-orientation pane) :vertical)
-        (make-space-requirement :min-width  minor :width  minor
-                                :min-height major :height major)
-        (make-space-requirement :min-width  major :width  major
-                                :min-height minor :height minor))))
+  (multiple-value-bind (vw vh)
+      (text-size pane (format-value (gadget-min-value pane) (slider-decimal-places pane)))
+    (setq vw
```

Comment:
`alexandria:maxf` can be used here.

Why is `vh` not updated in the same way?

Reply:
Yes, `maxf` is better. Updating `vh` seems like overkill, since there's only a difference for fonts that have digits of varying heights, it only affects the bounding box for horizontal sliders, and in any case the effect is minimal.

Reply:
is it costful though? if not, then having correct behavior is better.

Reply:
You're right. Commit https://github.com/McCLIM/McCLIM/pull/655/commits/a695dea5dc52727f1f6099f8fd622a3503981627 includes an improved `vw` and `vh` computation that is correct in all cases.

Reply:
this change is superseded. resolving this conversation.